go.mod

```diff
@@ -68,6 +67,8 @@ require (
 	github.com/hashicorp/golang-lru v0.5.1
 	github.com/hashicorp/hcl v1.0.0
 	github.com/hashicorp/nomad/api v0.0.0-20190412184103-1c38ced33adf
+	github.com/hashicorp/raft v1.0.2-0.20190617182316-3db06beda834
```

Comment:
@jefferai How does this work? I don't see v1.0.2 as a release in the raft repo. Does gomod create a pseudo release number for deps that are based off branches (Brian suggested using https://github.com/hashicorp/raft/pull/339 locally for now back when the `replace` directive was being used)? If so, why isn't it 1.1.x since 1.1.0 is the latest release?

Reply:
Because this wasn't against `master`, it was against a particular PR's branch, which is post-1.0.2 but pre-1.1.0.